example/react-native.config.js

```diff
@@ -1,13 +1,18 @@
 if (process.argv.includes("--config=metro.config.macos.js")) {
-  module.exports = {
-    reactNativePath: "node_modules/react-native-macos",
-  };
+    module.exports = {
```

Comment:
Can you reformat this file? You need to run `yarn` and `yarn prettier --write example/react-native.config.js`.